### DISCUSSION TITLE
Download (pdf + machine readable format) invoice

### DISCUSSION THREAD

**Discussion by mwosinski** [allegro/allegro-api#4622] [Suggestions - rejected]
Hi, 
I'm missing a functionality to automatically download invoices issued by allegro. It would be good to be able to download them both in human-readable (pdf) and machine-readable (xml/edi/...) format. This would help automate monthly accounting/billing processes - especially if managing multiple accounts.

**Comment by MartaNowaczyk** [comment]
Suggestion rejected. 
Reason: from 2024, the mandatory National e-Invoicing System comes into effect. This imposes an obligation on Allegro to issue invoices in a structured format and send it to this system in xml format.
